README.md

```diff
@@ -132,6 +132,10 @@ of any of the database's listed below to set up and initialize your database:
 
 [Instructions for running app locally](./docs/run_langchain_demo.md)
 
+### Clean up resources
+
+[Instructions for cleaning up resources](./docs/clean_up.md)
```

Comment:
Yes, the format is correct. Multiple clean up instructions might be too much since the clean up instructions is not too long. I can move the datastore specific clean up to their own setup docs (plus, mention to not clean up until the end of demo) and link it from the clean up doc.